one_fm/utils.py

```diff
@@ -898,38 +898,52 @@ def increase_daily_leave_balance():
     '''
 
     # Get List of Leave Allocation for today of a Leave Type (having Is Paid Annual Leave marekd True)
-    allocation_list = get_paid_annual_leave_allocation_list()
```

Comment:
Please you removed a block here, is this intentional ?

Reply:
Yes